rotkehlchen/db/schema.py

```diff
@@ -296,10 +298,44 @@
 );
 """
 
+DB_CREATE_AMM_TRADES = """
+CREATE TABLE IF NOT EXISTS amm_trades (
+    tx_hash VARCHAR[42] NOT NULL,
+    log_index INTEGER NOT NULL,
+    address VARCHAR[42] NOT NULL,
+    timestamp INTEGER NOT NULL,
+    location CHAR(1) NOT NULL DEFAULT('A') REFERENCES location(location),
+    type CHAR(1) NOT NULL DEFAULT ('A') REFERENCES trade_type(type),
+    is_base_asset_unknown INTEGER NOT NULL,
+    base_asset_address VARCHAR[42] NOT NULL,
+    base_asset_symbol TEXT NOT NULL,
+    base_asset_name TEXT,
+    base_asset_decimals INTEGER,
+    is_quote_asset_unknown INTEGER NOT NULL,
+    quote_asset_address VARCHAR[42] NOT NULL,
+    quote_asset_symbol TEXT NOT NULL,
+    quote_asset_name TEXT,
+    quote_asset_decimals INTEGER,
+    amount TEXT,
+    rate TEXT,
+    fee TEXT,
+    notes TEXT,
+    PRIMARY KEY (tx_hash, log_index)
+);
+"""
+
+DB_CREATE_INDEXES_AMM_TRADES = """
```

Comment:
Hey not a bad idea, but we don't use sqlite indices in rotki yet. We have not noticed any DB queries that impact performance yet.

I will probably delete them for now and when we decide to improve the DB performance we can identify bottlenecks and created indices for those tables then.